packages/discord.js/src/structures/interfaces/InteractionResponses.js

```diff
@@ -264,23 +263,6 @@ class InteractionResponses {
     this.replied = true;
   }
 
```

Comment:
Can't comment on the method for some reason (`sendPremiumRequired`), but there are 5 different places where this method is still in